index.js

```diff
@@ -15,11 +16,20 @@ var nameMap = {
 };
 
 module.exports = function (release) {
-	var version = /\d+\.\d+/.exec(release || os.release());
+	var verRe = /\d+\.\d+/;
+	var version;
 
-	if (!version) {
+	if (release || semver.gte(process.version, '3.1.0')) {
+		version = verRe.exec(release || os.release());
+	} else {
+		// Workaround for Windows 10 on node < 3.1.0
+		var execSync = require('child_process').execSync;
+		version = verRe.exec(String(execSync('ver.exe', {timeout: 1e5})));
```

Comment:
`execSync` doesn't work on Node 0.10, which unfortunately this module needs to support, can you use https://github.com/sindresorhus/exec-file-sync instead?